modules/sdk-coin-osmo/src/lib/index.ts

```diff
@@ -1,10 +1,6 @@
 import * as Constants from './constants';
 import * as Utils from './utils';
 
-export { OsmoTransaction } from './transaction';
-export { OsmoTransactionBuilder } from './transactionBuilder';
-export { TransactionBuilderFactory } from './transactionBuilderFactory';
-export { OsmoTransferBuilder } from './transferBuilder';
```

Comment:
Lets have the same transaction builder modules exported from cosmos abstract module for backward compatibility.